HomeEdu/Data/Source/Socket/HomeduSocket.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  HomeduSocket.swift
+//  HomeEdu
+//
+//  Created by Minh Mon on 7/31/19.
+//  Copyright © 2019 Nguyen Cuong. All rights reserved.
+//
+
+import Foundation
+import SocketIO
+import UserNotifications
+
+class HomeduSocket {
+    
+    static let shared = HomeduSocket()
+    
+    let manager = SocketManager(socketURL: URL(string: "http://homedu.herokuapp.com")!, config: [.log(true), .compress])
+    
+    /// Config HomeduSocket to listen event from server
+    func configSocket() {
+        let socket = manager.defaultSocket
+        
+        let center = UNUserNotificationCenter.current()
+        
+        socket.on(clientEvent: .connect) {data, ack in
```

Comment:
```suggestion
        socket.on(clientEvent: .connect) { data, ack in
```
lỗi coding convention, tương tự với các closure bên dưới